src/utils/consts.ts

```diff
@@ -25,6 +25,7 @@ export const VOICE_PLAYER_AUDIO_ID = 'sendbird-global-audio-player-id';
 // voice message file
 export const VOICE_MESSAGE_FILE_NAME = 'Voice_message.mp3';
 export const VOICE_MESSAGE_MIME_TYPE = 'audio/mp3;sbu_type=voice';
+export const VOICE_MESSAGE_MIME_TYPE__M4A = 'audio/x-m4a;sbu_type=voice';
```

Comment:
```suggestion
export const VOICE_MESSAGE_MIME_TYPE__XM4A = 'audio/x-m4a;sbu_type=voice';
```

Reply:
It would be good to explicitly use a x-m4a.

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/1123/commits/04f1d6b333c6e35a2739540c3f3ed5b50853422a